.github/workflows/ebpf.yml

```diff
@@ -116,6 +119,15 @@ jobs:
         Start-MpScan -ScanType QuickScan
         Add-MpPreference -ExclusionPath ${{ github.workspace }}
 
+    - name: Download GH CLI if not installed
```

Comment:
Why does this just now need to be added?

Reply:
Ah, I see you're using the cli below to find the artifact...

Reply:
Should I use https://github.com/marketplace/actions/get-latest-release instead? My only concern is taking dependencies on unknown actions.